html.go

```diff
@@ -106,12 +114,12 @@ const (
 // title is the title of the document, and css is a URL for the document's
 // stylesheet.
 // title and css are only used when HTML_COMPLETE_PAGE is selected.
-func HtmlRenderer(flags int, title string, css string) Renderer {
-	return HtmlRendererWithParameters(flags, title, css, HtmlRendererParameters{})
+func HtmlRenderer(flags int, title string, css string, sanitize SanitizedAnchorNameFunc) Renderer {
+	return HtmlRendererWithParameters(flags, title, css, sanitize, HtmlRendererParameters{})
 }
 
 func HtmlRendererWithParameters(flags int, title string,
-	css string, renderParameters HtmlRendererParameters) Renderer {
+	css string, sanitize SanitizedAnchorNameFunc, renderParameters HtmlRendererParameters) Renderer {
```

Comment:
Why did you choose to change the signature of `HtmlRendererWithParameters` instead of adding the `SanitizedAnchorNameFunc` as a new field to `HtmlRendererParameters`?